install.sh

```diff
@@ -30,6 +30,18 @@ else
     echo "Skip installing ROS"
 fi
 
+if [ "$ros2_distro" '==' "foxy" ]; then
+    # OpenVINO environment installation
+    echo -n "Do you want to install OpenVINO automatically? (y/N): "
```

Comment:
"Do you want to install Intel RealSense SDK and OpenVINO automatically?"
"Note that if you choose yes, it means you agree to the Intel license of the software."
"Install or not? (y/N): "

Reply:
The output message has been modified to requested content as image below.
![Screenshot from 2021-08-17 19-24-09](https://user-images.githubusercontent.com/72009899/129718956-4937ba8b-9927-4c2a-bd02-0a4a3a0d4bb9.png)